include/rmm/device_buffer.hpp

```diff
@@ -392,4 +392,29 @@ class device_buffer {
       mr::get_default_resource()};  ///< The memory resource used to
                                     ///< allocate/deallocate device memory
 };
+
+/**--------------------------------------------------------------------------*
+ * @brief Copies rmm::device_buffer to a preallocated host buffer.
+ *
+ * Copies device memory asynchronously on the specified stream
+ *
+ * @throws std::runtime_error if `hb` is `nullptr` or copy fails
+ *
+ * @param db `rmm::device_buffer` to copy to host
+ * @param hb host allocated buffer to copy data to
+ * @param stream CUDA stream on which memory may be allocated if the memory
```

Comment:
Previous doc seemed incomplete. 
```suggestion
 * @param stream CUDA stream on which the device to host copy will be performed
```

Reply:
Oops 😅 Thanks for catching that 🙂